checks/raw/sast.go

```diff
@@ -67,19 +66,19 @@ func SAST(c *checker.CheckRequest) (checker.SASTData, error) {
 	}
 	data.Workflows = append(data.Workflows, sonarWorkflows...)
 
-	snykWorkflows, err := getSnykWorkflows(c)
+	snykWorkflows, err := getSastUsesWorkflows(c, "^snyk/actions/.*", checker.SnykWorkflow)
 	if err != nil {
 		return data, err
 	}
 	data.Workflows = append(data.Workflows, snykWorkflows...)
 
-	pysaWorkflows, err := getPysaWorkflows(c)
+	pysaWorkflows, err := getSastUsesWorkflows(c, "^facebook/pysa-action$", checker.PysaWorkflow)
 	if err != nil {
 		return data, err
 	}
 	data.Workflows = append(data.Workflows, pysaWorkflows...)
 
-	qodanaWorkflows, err := getQodanaWorkflows(c)
+	qodanaWorkflows, err := getSastUsesWorkflows(c, "^JetBrains/qodana-action$", checker.QodanaWorkflow)
```

Comment:
Do we need to worry about case sensitivity here (or generally for any of them)?

Reply:
I'm not 100% sure, but as far as I can tell the answer is *yes* considering `uses` can refer to Docker container images https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses which have case sensitive tags.

Reply:
docker images also start with `docker://`, but we can start with case sensitivity until we see evidence otherwise.
https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-docker-hub-action

Reply:
I experimented a bit https://github.com/DavidKorczynski/intellij-ton/commit/b97abd243e910eba1bb562ad98e38f2d8da7284f and https://github.com/DavidKorczynski/intellij-ton/commit/2b1cc7cbe9c6abcdc6a9656a9cc32a5c4d9fcf45 -- and in this case the action is case insensitive. So, I'm kinda leaning towards there being no case sensitivity, but will just check up a bit more.